src/main/scala/com/codacy/clangtidy/CodacyReportSerializer.scala

```diff
@@ -0,0 +1,21 @@
+package com.codacy.clangtidy
+
+import java.nio.file.Path
+
+import com.codacy.model.ToolResults
+import io.circe.generic.AutoDerivation
+import io.circe.syntax._
+import io.circe.{Encoder, Printer}
+
+class CodacyReportSerializer extends AutoDerivation {
```

Comment:
Is there a reason for this being a class?

Reply:
Not really, it can be an object

Reply:
Fixed